filter/imagetopdf.c

```diff
@@ -997,6 +958,113 @@ main(int  argc,				/* I - Number of command-line arguments */
   colorspace = ColorDevice ? CUPS_IMAGE_RGB_CMYK : CUPS_IMAGE_WHITE;
 
   img = cupsImageOpen(filename, colorspace, CUPS_IMAGE_WHITE, sat, hue, NULL);
+  if(img!=NULL){
+
+  int margin_defined = 0;
+  int fidelity = 0;
+  int document_large = 0;
+
+  if(ppd->custom_margins[0]||ppd->custom_margins[1]||ppd->custom_margins[2]||ppd->custom_margins[3])
```

Comment:
@tillkamppeter Is this the right way to check for margins?

Reply:
I think not, AFAIR the custom_margins[] array is for the margins for a custom page size. If as output page size one of the list of the PageSize option is chosen, the margins assigned to this size (from the ImageableArea lines in the PPD, should be also somewhere in CUPS' ppd data structure) are the ones you are looking for.

Reply:
Thanks, @tillkamppeter. I have updated the PR.